### ISSUE TITLE
Water is not pouring

### ISSUE THREAD

**Issue by usach3v** [studio-minus/ppg-bugs-and-requests#849]
**Feature description**
So that the water does not flood, it's hard to explain, when you immerse the box with the person in the water, the water floods, can you solve it?
**Feature purpose**
Improved mechanics

**Additional context**

**Comment by 404CoderNotFound** [comment]
I don't think this is hard to explain, but you should just use translate. I don't understand what you're saying.

**Comment by mestiez** [comment]
I can't decipher this

**Comment by vda8** [comment]
he's basically saying "at water physics, no water physics is a bug."
